src/pages/settings/Wallet/AddDebitCardPage.js

```diff
@@ -94,6 +100,10 @@ function DebitCardPage(props) {
         return errors;
     };
 
+    if (!Permissions.canUseWallet(props.betas)) {
```

Comment:
Not A Blocker, but it would have been nice to add a comment explaining why this block is here. Perhaps you can do that in a followup PR?